.travis.yml

```diff
@@ -47,7 +47,7 @@ script:
     - docker images
     - ls -alF /home/travis/auth
     - docker pull alpine
-    - docker login localhost:5000 --username testuser --password testpassword
+    - docker login localhost:5000 --username testuser --password testpassword; y
```

Comment:
I think you will need to do something like
echo y | docker login localhost:5000 --username testuser --password testpassword